ci-operator/config/medik8s/self-node-remediation/medik8s-self-node-remediation-main__4.12.yaml

```diff
@@ -53,24 +43,41 @@ tests:
   skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
   steps:
     cluster_profile: medik8s-aws
-    dependencies:
-      OO_INDEX: ci-index-self-node-remediation-bundle
     env:
       BASE_DOMAIN: medik8s-ci.projects.ecosystem.sysdeseng.com
-      OO_CHANNEL: stable
-      OO_INSTALL_NAMESPACE: self-node-remediation
-      OO_PACKAGE: self-node-remediation
-      OO_TARGET_NAMESPACES: '!all'
+      INSTALL_NAMESPACE: self-node-remediation
     test:
-    - as: e2e
-      commands: export SKIP_LOG_VERIFICATION="true" SKIP_OOST_REMEDIATION_VERIFICATION="true"
-        && make e2e-test
-      from: src-oc
+    - as: install
+      cli: latest
+      commands: |
+        oc create namespace "$INSTALL_NAMESPACE"
+        operator-sdk run bundle -n "$INSTALL_NAMESPACE" quay.io/medik8s/self-node-remediation-operator-bundle:latest
```

Comment:
this line needs to be removed, it's installing SNR from quay, but we want to install the version built by CI in the line below

Reply:
🤦 thank you